libs/core/langchain_core/tracers/event_stream.py

```diff
@@ -244,6 +288,8 @@ async def on_llm_new_token(
 
         if run_info is None:
             raise AssertionError(f"Run ID {run_id} not found in run map.")
+        if self.is_tapped.get(run_id):
```

Comment:
avoid duplicate stream events when chat model is tapped (which could already happen before this pr)